Classes/gitx_askpasswd_main.m

```diff
@@ -171,7 +171,10 @@ int	main( int argc, const char** argv )
 		
 		[defaults synchronize];
 		
-		return code;
+		if (code <= INT_MAX) {
+			return (int)code;
+		};
+		return INT_MAX;
```

Comment:
[`runModalForWindow:`][1] only returns [`NSModalResponse`][2] values, so the bounds checking isn't needed, but this should probably just return 1 (`EXIT_FAILURE`) on error (`NSModalResponseAbort`?).

[1]: https://developer.apple.com/reference/appkit/nsapplication/1428436-runmodalforwindow?language=objc
[2]: https://developer.apple.com/reference/appkit/nsmodalresponse?language=objc